app/components/listing/LocationTimesAccordion.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { LocationDetails } from "models";
+import classNames from "classnames";
+import styles from "./LocationTimesAccordion.module.scss";
+
+/*
+    Accordion built per a11y guidelines:
+    https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
+*/
```

Comment:
Thanks for this. I'd love to chat more about how me might add additional analysis and certification to our components. So we might certify a component through a third-party check of some kind and be able to label it here as like _Meets certification requirements. Evaluated by XYZ checker thingy._

Reply:
We can run our staging site through the WAVE AIM tool as part of a11y QA?